Project.toml

```diff
@@ -16,35 +16,31 @@ Requires = "ae029012-a4dd-5104-9daa-d747884805df"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [weakdeps]
```

Comment:
Do we still need these weak deps?

Reply:
I don't think we do given all the extensions are gone.